src/administration/integrations/webhooks.md

```diff
@@ -11,7 +9,71 @@ launch automated user testing.
 $ platform integration:add --type=webhook --url=A-URL-THAT-CAN-RECEIVE-THE-POSTED-JSON
 ```
 
-Here you can see an example of the response:
+The webhook URL will receive a POST message for every "Activity" that is triggered, and the message will contain complete information about the entire state of the project at that time.  In practice most of the message can be ignored but is available if needed.  The most commonly used values are documented below.
+
+It's also possible to set the integration to only send certain activity types, or only activities on certain branches.  The CLI will prompt you to specify which to include or exclude.  Leave at the default values to get all events on all environments in a project.
+
+## Webhook schema
+
+### `id`
+
+A unique opaque value to identify the activity.
+
+### `project`
+
+The Project ID for which the activity was triggered.  Use this value if you want to have multiple projects POST to the same URL.
+
+### `type`
+
+The `type` property specifies the event that happened.  Its value is one of:
+
+* `environment.push`: A user has pushed code to a branch, either existing or new.
+* `environment.branch`: A new branch has been created via the UI. (A branch created via a push will show up only as an `environment.push`.)
+* `environment.backup`: A user triggered a [snapshot](/administration/snapshot-and-restore.md)
+* `environment.restore`: A user triggered a [snapshot](/administration/snapshot-and-restore.md)
+* `environment.subscription.update`: I don't know what this is.
```

Comment:
Please advise what this one is.  I think it's a billing update?

Reply:
Yes, it's "the master environment has been resized because the subscription has changed".

Reply:
Will update.